src/test/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePassTest.scala

```diff
@@ -219,31 +223,44 @@ class XMLPropertyTests extends PropertiesFilePassTestBase(".xml") {
 /** Base class for tests on properties files and Java code.
   */
 // file extension to support for any file as properties
-abstract class PropertiesFilePassTestBase(fileExtension: String)
+abstract class PropertiesFilePassTestBase(fileExtension: String, language: String = "java")
```

Comment:
As this base class is used across language, this should not be a part of the test java/passes.

Externalize this base class